scripts/general.js

```diff
@@ -10,4 +10,13 @@
 // Author:
 //   YashdalfTheGray
 
-module.exports = (robot) => {}; // eslint-disable-line no-unused-vars
+module.exports = (robot) => {
+  robot.hear(/^randbot:? (.+)/i, (res) => {
+    res.reply([
+      'That was my old form.',
+      'I have become perfect.',
+      'I am become RoboTK.',
+      'You shall address me as such.'
```

Comment:
NB I almost feel like:

“Randbot.... that is a name I haven’t heard in a long time”

Or 

“When I met you I was but a learner, now I am the master”